### ISSUE TITLE
Add parameter biplot visualisations

### ISSUE THREAD

**Issue by jonc125** [ModellingWebLab/WebLab#20]
e.g. figs 4 & 5 in http://rsos.royalsocietypublishing.org/content/2/12/150499.figures-only, or the more heat map style in Aidan's thesis.

Modellers (@mirams, @michaelclerx), any preferences on what these should look like, and what a sensible UI would be for selecting what to plot?

**Comment by mirams** [comment]
Your Figure 4 looks nice, it would be good to be able to do these:

![f4 large](https://user-images.githubusercontent.com/858776/31957147-b89f63ee-b8e5-11e7-953b-f76d61ad0dec.jpg)

For very converged MCMC chains with loads of samples though you would end up with too many red blobs on top of one another to get an idea of the density. Ross has made some lovely matrix plots a bit like these where they turn into heatmaps with high numbers of points, Chon and Michael have access to the matplotlib for them I think. Here's an example:
![model_8_mcmc_noisy_adaptive_covariance_matrix_protocol_6_c_seed_1_scatterplot_matrix_normalized_colormap_subset](https://user-images.githubusercontent.com/858776/31957051-4ccbbcda-b8e5-11e7-843b-794be4ca10ab.png)

In terms of the UI, I'm not sure, presumably it becomes part of the list of plots from the fitting description?

**Comment by MichaelClerx** [comment]
I agree with Gary! But ideally you'd be able to do the top one for short chains and the bottom one for long chains.

I've experimented with figures like the top one a bit and you run into two problems: 1. The files can get unusually large, causing issues for emails and online submission systems and such (in one case prompting a publisher to create an ugly pdf from our eps without telling us) 2. It's very tempting to use transparency to get beautiful plots, but lots of viewing and printing software doesn't fully support this leading to very ugly plots
